openshift/rh-che.config.yaml

```diff
@@ -71,3 +70,5 @@ data:
   CHE_WORKSPACE_STORAGE: "/home/user/che/workspaces"
   CHE_WORKSPACE_PLUGIN__BROKER_VSCODE_IMAGE: "eclipse/che-vscode-extension-broker:v0.13.0"
   CHE_FABRIC8_CDN_PREFETCH_EDITOR: 'NULL'
+  che.core.jsonrpc.processor.major.max.pool.size: 120
```

Comment:
Would be great to have some clarity on why this value is used. With default minor endpoint pool size equal to 100 we have 220 threads overall for 2 endpoints. I'm not sure how to evaluate whether our RAM size for Che is enough for 220 threads or not. Do you have any hints on how to evaluate that?

Reply:
This is major endpoint setting explicitly. The minor will be set to much lower value once I complete the testing on beta-2. I have set that value that high, because the threads don't consume that much ram and where ineffective at consuming the messages in queue when set to lower values.